test/e2e/e2e_setup_test.go

```diff
@@ -182,6 +184,10 @@ func Setup(specName string, f func(testSpecificSettings func() testSettings), op
 			OutputPath:           testSpecificClusterctlConfigPath,
 			Variables:            testSpecificVariables,
 		})
+
+		if testMode == SupervisorTestMode {
+			runtimeExtensionProviders = []string{"vm-operator", "net-operator"}
+		}
```

Comment:
```suggestion
		if testMode == SupervisorTestMode {
			runtimeExtensionProviders = []string{"vm-operator"}
			if testTarget == VCSimTestTarget {
				runtimeExtensionProviders = append(runtimeExtensionProviders, "net-operator")
			}
		}
```

Reply:
net-operator is only needed when using vcsim.

Reply:
Fixed!